### DISCUSSION TITLE
Creating a new comment component for mdx

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#945] [Q&A] [answered]
Hey this is an idea for creating a component for the mdx file in short words. 

What this will do. 

Currently to write comments or comment some lines or words using `ctrl + /` we need to use ```{/*  */} ``` and it takes a lot of time. to do that multiple time. 

Instead if we just do `Ctrl + /`  the code is commented in mdx using `<!-- -->` it throws an error as `!` cannot be used. So I am thinking of creating or changing some things in this so that when there is `<!-- -->` in the mdx file it will be wrapped using `{/*  */}`. It will also be good in the future as it will work normally. 

I think the error is thrown due to `!` while the code cannot be commented because this is being converted into typescript or javascript component in which comment is only possible via ```{/*  */} ```, so is there any way we can create or do this ?  

I don't know if nextra supports this type of comments or not, but if it does I would like to add this same feature to fumadocs. 

Is there any way.

**Comment by fuma-nama** [comment, marked as answer]
You can propose the change to MDX.js, we strictly allow the syntax of MDX to reduce maintenance costs and ensure it's easy to begin for writers who is experienced with markdown/MDX from other frameworks.
